port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -118,12 +124,20 @@ async def _search_as_object(
         for key, task in search_tasks.items():
             try:
                 if isinstance(task, list):
-                    result[key] = [await task for task in task]
+                    result_list = []
+                    for task in task:
+                        task_result = await task
+                        if task_result is None:
+                            misconfigurations[key] = obj[key]
```

Comment:
wouldn't it fail? if `misconfigurations` object wasn't passed or passed as `None`? I couldn't find where you initialize it inside of the method while you are supporting `None`

Reply:
great catch! thank you 🙏

Reply:
its being initialized at line 152 and passed into the method, but in any case, this should be covered and if None it should be initialised to prevent an error.